packages/editor/src/extensions/react/react-node-view.tsx

```diff
@@ -168,7 +168,8 @@ export class ReactNodeView<P extends ReactNodeViewProps> implements NodeView {
               attr,
               this.getPos(),
               options?.addToHistory,
-              options?.preventUpdate
+              options?.preventUpdate,
+              options?.forceUpdate
```

Comment:
You need to make these same changes in `/src/extensions/react/selection-based-react-node-view.tsx` as well.